curator/run_cron.py

```diff
@@ -107,9 +126,15 @@
             base_cmd = '/usr/bin/curator --loglevel ' + curlvl + ' ' + connection_info + ' ' + operation + ' indices --timestring %Y.%m.%d'
             tab_command = base_cmd + ' --older-than ' + str(value) + ' --time-unit ' + unit
 
+            # construct regex to match all projects for this op/time/unit
+            tab_command = tab_command + ' --regex '
+            regex = '('
+            sep = ''
             for project in curator_settings[operation][unit][value]:
-                tab_command = tab_command + ' --prefix ' + project + '.'
-
+                regex = regex + sep + '^' + re.escape(project + '.')
+                sep = '|'
+            regex = regex + ')'
+            tab_command = tab_command + shellquote(regex)
```

Comment:
this works ok but it feels like it ought to be reduced to a one-liner, something like (not a pythonista):

    tab_command = tab_command + ' --regex ' + shellquote('(' + '|'.join(map(lambda project:'^'+re.escape(project+'.'),curator_settings[operation][unit][value] )) + ')')

OK, maybe two lines, or to taste. I leave it up to you though.